src/index.html

```diff
@@ -7,6 +7,431 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="header">
+        <a href="/" class="logo-img">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <nav>
+          <ul class="nav">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#">
+                Apple
+              </a>
+            </li>
+    
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Samsung
+              </a>
+            </li>
+    
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Smartphones
+              </a>
+            </li>
+    
+            <li class="nav__item">
+              <a 
+                class="nav__link" 
+                href="#" 
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+    
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Gadgets
+              </a>
+            </li>
+    
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Tablets
+              </a>
+            </li>
+    
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Photo
+              </a>
+            </li>
+    
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="container">
+        <div class="card" data-qa="card">
+          <img 
+            src="./images/imac.jpeg" 
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)" 
+            class="card__img"
+          >
+        
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+        
+          <p class="card__text-code">
+            Product code: 195434
+          </p>
+        
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__item "></div>
+              <div class="stars__item "></div>
+              <div class="stars__item "></div>
+              <div class="stars__item "></div>
+              <div class="stars__item "></div>
+            </div>
+        
+            Reviews: 5
+          </div>
+        
+          <div class="card__price price">
```

Comment:
why do we need to create separate BEM block `price`?

Reply:
not fixed
<img width="417" alt="image" src="https://user-images.githubusercontent.com/53611910/216793774-b7f09136-1a55-47c5-b38a-48f731590c91.png">